docs/aca/02-aca-comm/index.md

```diff
@@ -399,8 +399,8 @@ So far the Frontend App is sending HTTP requests to publicly exposed Web API, an
     az containerapp update `
     --name "$FRONTEND_WEBAPP_NAME"  `
     --resource-group $RESOURCE_GROUP `
-    --set-env-vars "BackendApiConfig__BaseUrlExternalHttp=https://tasksmanager-backend-api.internal.[Environment unique identifier].eastus.azurecontainerapps.io"
```

Comment:
I don't believe the change is required. If you look at (this link)[https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-7.0#key-per-file-configuration-provider] you will notice that they mention how the double-underscore (__) is used as a configuration key delimiter in file names. I also confirmed that its working on my side with double-underscore. Are you suggeting that it didn't work for you with double-underscores?

Reply:
Hello @pankajagrawal16 Thanks for your feedback. 

The double underscore is needed here, reason for this than Colon (:) separators aren't supported in environment variable names. We must use a double underscore (__) in place of a colon. The [Environment Variables configuration provider](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-7.0#environment-variables) converts double-underscores into colons when environment variables are read into configuration. In our example, the env var key `BackendApiConfig:BaseUrlExternalHttp` is set to `BackendApiConfig__BaseUrlExternalHttp`

You can read more [here](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/linux-nginx?view=aspnetcore-7.0&tabs=linux-ubuntu#create-the-service-file) I will close the PR now as there is no need to change it.

FYI @waelkdouh